src/Compilers/CSharp/Portable/Symbols/SymbolEqualityComparer.cs

```diff
@@ -40,6 +41,29 @@ private SymbolEqualityComparer(TypeCompareKind comparison)
             _comparison = comparison;
         }
 
+        internal static EqualityComparer<Symbol> Create(TypeCompareKind comparison)
```

Comment:
>EqualityComparer<Symbol> [](http://example.com/codeflow?start=24&length=24)

```SymbolEqualityComparer```?

Reply:
No need. All the pre-baked instances are `EqualityComparer<Symbol>` too. Consumers don't need to know the specific type